src/services/local_storage.service.ts

```diff
@@ -0,0 +1,73 @@
+import {Injectable} from '@angular/core';
+
+const DEFAULT_TIME = 7 * 24 * 60 * 60 * 1000; // 7 days * 24 hours/day * 60 min/hour * 60 s/min * 1,000 ms/s
+const ACCESS_TOKEN = 'access_token';
+const IS_ADMIN = 'is_admin';
+
+@Injectable()
+export class LocalStorageService {
+
+  constructor() {}
+
+  public static addAccessToken(token: string, time_to_live?: number): void {
+    if (!time_to_live) {
+      time_to_live = DEFAULT_TIME;
+    }
+    LocalStorageService.addItem(token, ACCESS_TOKEN, time_to_live);
+  }
+
+  public static removeAccessToken(): void {
+    localStorage.removeItem(ACCESS_TOKEN);
+  }
+
+  public static getAccessToken(): string {
+    return LocalStorageService.getItem(ACCESS_TOKEN);
+  }
+
+  public static addIsAdmin(isAdmin: boolean, time_to_live?: number): void {
+    let string = 'false';
+    if (isAdmin === true) {
+      string = 'true';
+    }
+    if (!time_to_live) {
+      time_to_live = DEFAULT_TIME;
+    }
+    LocalStorageService.addItem(string, IS_ADMIN, time_to_live);
+  }
+
+  public static removeIsAdmin(): void {
+    localStorage.removeItem(IS_ADMIN);
+  }
+
+  public static getIsAdmin(): boolean {
+    const result = LocalStorageService.getItem(IS_ADMIN);
+    if (!result) {
+      return null;
+    } else {
+      return result === 'true';
+    }
+  }
+
+  private static addItem(item: string, label: string, time_to_live: number): void {
```

Comment:
You can make item of type any, `item: any`. Doing this will allow this service to add any type of object to local storage, and when you get it back out, it will be of that type. Saving everything as a string means that if you wanted to add an object, you'd have to stringify it before and after adding it and after getting it back.

Reply:
Also change `time_to_live` to `timeToLive` to match convention.

Reply:
Nice, I wasn't aware of the `any` type. That will make things much cleaner!